templates/zerver/emails/confirm_registration.txt

```diff
@@ -3,7 +3,4 @@
 {{ _('Click the link below to complete registration.') }}
     <{{ activate_url }}>
 
-{% trans %}
-Contact us any time at {{ support_email }} if you run into trouble,
-have any feedback, or just want to chat!
-{% endtrans %}
+{% trans %}Contact us any time at {{ support_email }} if you run into trouble,have any feedback, or just want to chat!{% endtrans %}
```

Comment:
There is an extra space at the end of `chat!` here?

Reply:
I don't think so. `or just want to chat!{% endtrans %}` copied straight from the file.